src/main/java/org/folio/rest/impl/PostGobiOrdersHelper.java

```diff
@@ -41,18 +21,30 @@
 import org.folio.rest.gobi.model.ResponseError;
 import org.folio.rest.jaxrs.model.Mapping;
 import org.folio.rest.jaxrs.model.OrderMappings;
+import org.folio.rest.tools.utils.BinaryOutStream;
 import org.w3c.dom.Document;
 import org.w3c.dom.Node;
 
-import io.vertx.core.AsyncResult;
-import io.vertx.core.Context;
-import io.vertx.core.Future;
-import io.vertx.core.Handler;
-import io.vertx.core.http.HttpMethod;
-import io.vertx.core.json.JsonObject;
+import javax.ws.rs.core.Response;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathConstants;
+import javax.xml.xpath.XPathFactory;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.folio.gobi.LookupService.CONFIGS;
+import static org.folio.gobi.LookupService.FIRST_ELEM;
+import static org.folio.gobi.LookupService.QUERY;
+import static org.folio.rest.ResourcePaths.CONFIGURATION_ENDPOINT;
+import static org.folio.rest.ResourcePaths.ORDERS_BY_ID_ENDPOINT;
+import static org.folio.rest.ResourcePaths.ORDERS_ENDPOINT;
+import static org.folio.rest.jaxrs.resource.Gobi.PostGobiOrdersResponse.respond400WithApplicationXml;
+import static org.folio.rest.jaxrs.resource.Gobi.PostGobiOrdersResponse.respond401WithTextPlain;
+import static org.folio.rest.jaxrs.resource.Gobi.PostGobiOrdersResponse.respond500WithApplicationXml;
 
 public class PostGobiOrdersHelper {
-  private static final Logger logger = LogManager.getLogger(PostGobiOrdersHelper.class);
+  private static final Logger LOGGER = LogManager.getLogger(PostGobiOrdersHelper.class);
```

Comment:
In all othe places logger defined with lower case? In this case I think it does not makes sense convention to full module